org.adempiere.base/src/org/compiere/model/MInOutLineConfirm.java

```diff
@@ -182,6 +182,11 @@ protected boolean beforeDelete ()
 	 */
 	protected boolean beforeSave (boolean newRecord)
 	{
+		MInOutConfirm parent = new MInOutConfirm(getCtx(), getM_InOutConfirm_ID(), get_TrxName());
+		if (newRecord && parent.isComplete()) {

```

Comment:
this is now inconsistent with isProcessed() check in other model class

Reply:
solved